app/(Home)/home/page.tsx

```diff
@@ -1,13 +1,16 @@
 import { getSession } from "../../actions/auth-action";
-import Link from "next/link";
+// import Link from "next/link";
 import React from "react";
-import { Button } from "@components/ui/Button";
-import Image from "next/image";
-import homeside from "@public/assets/images/homeside.png";
-import homefirst from "@public/assets/images/homefirst.png";
-import homesecond from "@public/assets/images/homesecond.png";
-import homethird from "@public/assets/images/homethird.png";
-import near from "@public/assets/images/near.png";
+// import { Button } from "@components/ui/Button";
```

Comment:
code not in use should be deleted, especially for files like the homepage that has a complete overhaul

Reply:
The page is looking good. I love how you made use of components 💯